src/Bubble.js

```diff
@@ -60,7 +60,7 @@ export default class Bubble extends React.Component {
     if (this.props.renderTicks) {
         return this.props.renderTicks(currentMessage);
     }
-    if (currentMessage.user._id !== this.props.user._id) {
+    if (currentMessage.fromSystem || (currentMessage.user._id !== this.props.user._id)) {
```

Comment:
Only as suggestion, maybe is better to extract `(currentMessage.user._id !== this.props.user._id)` to an `utils` function such as `isOwnMessage`. This function could be something like 

```js
function isOwnMessage(props) {
  const { currentMessage, user } = props;
  
  return currentMessage.user._id == user._id;
}
````

And then simply the condition to a

`if (currentMessage.fromSystem || !isOwnMessage(this.props)`

or even create a `isFromOtherUser` method in order to avoid the negation (`!`) inside the conditional.